compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -43,26 +43,29 @@ private[repl] class Rendering(compiler: ReplCompiler,
       myClassLoader
     }
 
-  /** Load the value of the symbol using reflection
+  /** Load the value of the symbol using reflection.
    *
    *  Calling this method evaluates the expression using reflection
    */
   private[this] def valueOf(sym: Symbol)(implicit ctx: Context): Option[String] = {
     val defn = ctx.definitions
-    val objectName = sym.owner.fullName.encode.toString.dropRight(1) // gotta drop the '$'
+    val objectName = sym.owner.fullName.encode.toString.stripSuffix("$")
     val resObj: Class[_] = Class.forName(objectName, true, classLoader())
-
-    val res =
+    val value =
       resObj
-        .getDeclaredMethods.find(_.getName == sym.name.toString + "Show").get
-        .invoke(null).toString
-
+        .getDeclaredMethods.find(_.getName == sym.name.encode.toString).get
```

Comment:
Does this assume a getter? Do we generate a getter for case like this:
```scala
> private[this] val foo = 1
```

Reply:
But I guess it doesn't hurt to avoid the .get...

Reply:
I was wondering if you could access the field instead